### ISSUE TITLE
Allow input commands to be canceled with ESC instead of empty input

### ISSUE THREAD

**Issue by LanHikari22** [anufrievroman/calcure#56]
Currently, if you press e to edit an event name, you have to input some empty string for it to cancel out. Same for the date inputs, event id input, new title input. These command inputs should recognize escape and just cancel.

**Comment by anufrievroman** [comment]
I agree it would be nice, but as far as I can see standard `curses` library doesn't allow that, you sort of must press Return to finish your input. Like I said in another thread, it seems that only switching to other libraries would fix that.
